app.rb

```diff
@@ -0,0 +1,131 @@
+require_relative './rental'
+require_relative './students'
+require_relative './books'
+require_relative './person'
+require_relative './teacher'
+require_relative './createstudents'
+require_relative './createteachers'
+require_relative './main'
+require_relative './listrents'
+require_relative './rent_much_book'
```

Comment:
- [ ] The file in your project is named `rent_books` and not `rent_much_book`, it's throwing an error when I run the app, kindly fix this issue 👍